README.md

```diff
@@ -4,6 +4,31 @@ A Buildkite library plugin with common utilities used in CI.
 
 ## Utilities
 
+### publish-languages
+
+Publish languages.json file to EventBridge
+
+```yml
+steps:
+  - command: publish-languages event_bus_name service_name user_id event_detail_type
+    plugins:
+      - tapendium/tap-ci-utils#v0.4.1: ~
```

Comment:
**nit:** Should this be minor version bump rather than a patch because it's new functionality?

Reply:
good point! 583c90b